### ISSUE TITLE
Detailed Calendar View (VueCalendar.lava) Shows AM for every event

### ISSUE THREAD

**Issue by cmronk** [BEMADEV/Room-Management#18]
### Please go through all the tasks below

- [X] Check this box only after you have successfully completed both the above tasks

### Please provide a brief description of the problem. Please do not forget to attach the relevant screenshots from your side.

On our particular Rock instance, all of the events within the Detailed Calendar (Vue.js) are showing as "AM" regardless of whether they are AM/PM
![Screenshot 2023-10-20 at 11 47 39 AM](https://github.com/BEMADEV/Room-Management/assets/35753005/755809f4-be1c-4c96-bf7c-d3169398ea22)

### Expected Behavior

We would expect to see differentiation between AM/PM Events 

### Actual Behavior

All Events show AM

### Steps to Reproduce

- Navigate to Reservation Management
- Select "Detailed Calendar" from "View As:" dropdown.
- Refresh to get past error (separate issue, seems to read the information inside the <v></v> as HTML instead of Vue)
![Screenshot 2023-10-20 at 2 36 29 PM](https://github.com/BEMADEV/Room-Management/assets/35753005/eb3c553c-3e7c-406a-8604-46a920870101)

Workaround - edit the date formats in the Lava.
I did actually go in the lava file and modify the date formats to include 'tt'.
`{% for reservationDate in ReservationDates %}
    {% for reservationSummary in reservationDate.Reservations %}
        {% assign adminContact = reservationSummary.AdminContactPersonAlias.Person %}
            {
                approvalState:"{{reservationSummary.ApprovalState }}",
                eventContact:'{{eventContact.FullName}}',
                eventContactId:{{eventContact.Id}},
                reservationStart:"{{reservationSummary.ReservationStartDateTime | Date:'M/d h:mm tt'}}",
                reservationEnd:"{{reservationSummary.ReservationEndDateTime | Date:'M/d h:mm tt'}}",
                eventStart:"{{reservationSummary.EventStartDateTime | Date:'M/d h:mm tt'}}",
                eventEnd:"{{reservationSummary.EventEndDateTime | Date:'M/d h:mm tt'}}",
                locations:"{{reservationSummary.Locations | Select:'Location' | Select:'Name' | Join:', ' | Replace:'"','\"'}}",
                resources:"{{reservationSummary.Resources | Select:'Location' | Select:'Name' | Join:', ' | Replace:'"','\"'}}",
                id:{{reservationSummary.Id}},
                name:"{{reservationSummary.ReservationName | Replace:'"','\"' }}",
                color:"{% case reservationSummary.ApprovalStateInt %}{% when 2 %}#16c98d{% when 3 %}#d4442e{% when 4 %}#ee7624{% when 1 or 5 or 6 %}#ffc870{% else %}#009ce3{% endcase %}",
                start:"{{reservationSummary.ReservationStartDateTime | Date:'yyyy-MM-ddTHH:mm'}}",
                end:"{{reservationSummary.ReservationEndDateTime | Date:'yyyy-MM-ddTHH:mm'}}",
                timed: true
            },
    {% endfor %}
{% endfor %}`

### Rock Version

15.2

### Plugin Version

2.4.1.14

### Client Culture Setting

en-US

**Comment by tcavaletto** [comment]
Included in Release 2.4.3.14
